bin/firefox-driver.js

```diff
@@ -43,4 +51,21 @@ if (shouldStart) {
   setInterval(() => {}, 100);
 }
 
+function getResults(driver) {
+  driver
+    .findElement(By.id("mocha-stats"))
+    .getText().then(results => {
+      console.log("results ", results);
+      const match = results.match(/failures: (\d*)/);
+      const resultCode = parseInt(match[1], 10) > 0 ? 1 : 0;
+      process.exit(resultCode);
+    });
+}
+
+if (isTests) {
+  const driver = start();
+  driver.get("http://localhost:8003");
+  setTimeout(() => getResults(driver), 5000);
```

Comment:
Is there a way we can wait for something to appear in the DOM instead of a setTimeout? This is going to introduce flakiness, for sure :)